pages/trainers.html

```diff
@@ -8,13 +8,16 @@ <h2>Melanie Ahn, 2nd Dan holder</h2>
 	Ze is begonnen met Taekwondo in 2002, en heeft de eerste Dan behaald in 2007. De tweede Dan volgde in 2009. Ze is begonnen met lesgeven aan kinderen van 4 tot 14 toen ze zelf 15 was. Ze geeft trainingen op donderdagen bij Ilyeo.
 </p>
 <p>
-	Ik hoop dat jullie evenveel plezier en zelfverzekerdheid uit taekwondo halen als ik.
+	"Ik hoop dat jullie evenveel plezier en zelfverzekerdheid uit taekwondo halen als ik."
 </p>
 
-<img src="images/trainers/ab.jpg" alt="Abdenasser Eddeane" width=589/>
-<h2>Abdenasser Eddeane, 2nd Dan holder</h2>
+<img src="images/trainers/tung.jpg" alt="Le Xuan Tung" width=589/>
+<h2>Le Xuan Tung, 2nd Dan holder</h2>
 <p>
-	Ab heeft een trainers licentie van niveau 3, die word erkent door de NOC / NSF, het Nederlanse Olympische Comité.
-	Ab begon Taekwondo toen hij 10 jaar was, en behaalde de eerste Dan op zijn 13de. Een jaar later, op zijn 14de, behaalde hij de 2de Dan.
-	Hij houd van Taekwond, en is vooral gepassioneerd over sparren. Hij heeft mee gedaan in nationale en internationale toernooien. Bij Ilyeo is hij verantwoordelijk voor het verbeteren van het spar-niveau, op fysieke en technische aspecten. Ab geeft de trainingen op vrijdag.
+	"Ik ben begonnen met Taekwondo bij Ilyeo in 2011, haalde de 1ste Dan in 2016 en haalde de 2de Dan in 2018.
+	Ik houd van Taekwondo door de snelle, krachtige en efficiënte bewegingen."
+	Hij geeft trainingen op maandagen en vrijdagen bij Ilyeo.
```

Comment:
Volgorde van quotes is een beetje verwarrend imo. Ik denk dat we het beter meer verhalend kunnen doen met op het einde een "motivational quote"?